lighthouse-core/audits/script-treemap-data.js

```diff
@@ -241,20 +243,19 @@ class ScriptTreemapDataAudit extends Audit {
           const sourceData = {
             resourceBytes: bundle.sizes.unmappedBytes,
           };
-          if (unusedJavascriptSummary.sourcesWastedBytes) {
+          if (unusedJavascriptSummary?.sourcesWastedBytes) {
             sourceData.unusedBytes = unusedJavascriptSummary.sourcesWastedBytes['(unmapped)'];
           }
           sourcesData['(unmapped)'] = sourceData;
         }
 
-        node = this.makeScriptNode(scriptElement.src, bundle.rawMap.sourceRoot || '', sourcesData);
+        node = this.makeScriptNode(script.name, bundle.rawMap.sourceRoot || '', sourcesData);
       } else {
         // No valid source map for this script, so we can only produce a single node.
-
         node = {
           name,
-          resourceBytes: unusedJavascriptSummary.totalBytes,
-          unusedBytes: unusedJavascriptSummary.wastedBytes,
+          resourceBytes: unusedJavascriptSummary?.totalBytes ?? script.length ?? 0,
```

Comment:
unusedJavascriptSummary.totalBytes and script.length should, if they both exist, always be equal.

but I figure preference should be given to the size as determined by the coverage data, if we are to display a value for percent wasted.